lib/vdsm/tool/configurators/multipath.py

```diff
@@ -109,3 +106,27 @@ def isconfigured():
 
     sys.stdout.write("multipath requires configuration\n")
     return NO
+
+
+def isconfigured():
+    """
+    Check the multipath daemon configuration.
+    """
+    conf_metadata_ok = _check_mpath_metadata()
+    try:
+        mpathconf.check_local_config()
+    except exception.InvalidConfiguration as e:
+        ufn_enabled_section = e.context["section"]
+        sys.stdout.write(
+            f"Invalid configuration detected: {e.context['reason']}:\n")
+        sys.stdout.write(f"  {ufn_enabled_section.name} {{\n")
+        for attr in ufn_enabled_section.attrs:
+            line_indent = "    "
+            if attr.key == "user_friendly_names":
+                line_indent = "->" + line_indent
```

Comment:
I don't think we need this since we start the warning with "user_friendly_names is enabled". Showing the config as is simpler and better.